pkg/tanka/static.go

```diff
@@ -15,22 +15,31 @@ import (
 type StaticLoader struct{}
 
 func (s StaticLoader) Load(path string, opts JsonnetOpts) (*v1alpha1.Environment, error) {
-	root, base, err := jpath.Dirs(path)
+	config, err := Peek(path, opts)
 	if err != nil {
 		return nil, err
 	}
 
-	config, err := parseStaticSpec(root, base)
+	data, err := EvalJsonnet(path, opts)
 	if err != nil {
 		return nil, err
 	}
 
-	data, err := EvalJsonnet(path, opts)
+	if err := json.Unmarshal([]byte(data), &config.Data); err != nil {
+		return nil, err
+	}
+
+	return config, nil
+}
+
+func (s StaticLoader) Peek(path string, opts JsonnetOpts) (*v1alpha1.Environment, error) {
```

Comment:
It feels strange that this function gets JsonnetOpts but doesn't do anything with it.

Reply:
The interface is generalized across all possible implementations. That the static implementation does not utilize Jsonnet is a lucky corner-case with performance benefits